database/create_tables.sql

```diff
@@ -39,35 +40,31 @@ CREATE TABLE Transactions (
 );
 
 CREATE TABLE Creates (
-    project_id int REFERENCES Projects(project_id),
+    project_name int REFERENCES Projects(project_name),
     user_id int REFERENCES Users(user_id),
+    create_date datetime NOT NULL,
     CONSTRAINT creates_constraint PRIMARY KEY(user_id, project_id)
 );
 
 CREATE TABLE SearchHistory (
-    search_id serial PRIMARY KEY,
+    search_timestamp timestamp,
+    search_text varchar(255),
+    CONSTRAINT searchhistory_timestamp PRIMARY KEY(search_timestamp)
+);
+
+CREATE TABLE Searches (
+    search_timestamp timestamp REFERENCES SearchHistory(search_timestamp),
     user_id int REFERENCES Users(user_id),
-    search_text varchar(255)
+    CONSTRAINT searches_constraint PRIMARY KEY(user_id, search_timestamp)
 );
 
-CREATE TABLE PaymentInfo (
-    paymentInfo_id serial PRIMARY KEY,
+CREATE TABLE Wallets (
     user_id int REFERENCES Users(user_id) ON DELETE CASCADE,
-    credit_card_number varchar(16),
-    credit_card_expiry_month int,
-    credit_card_expiry_year int,
-    cv_number varchar(3),
-
-    CONSTRAINT
-      credit_card_expiry_month
-      CHECK (credit_card_expiry_month BETWEEN 1 AND 12),
-    CONSTRAINT
-      credit_card_expiry_year
-      CHECK (credit_card_expiry_year > (extract(epoch from date_trunc('year', CURRENT_DATE))::int))
+    amount numeric,
+    CONSTRAINT wallets_constraint PRIMARY KEY(user_id, amount)
```

Comment:
Can the primary key be just the Foreign key? Hahah cause the amount feels abit redundant.

Reply:
But then what will the primary key for `Wallets` be? Unless we make `Wallets` a weak entity set.